ProcessLib/HeatTransportBHE/BHE/BHEAbstract.h

```diff
@@ -28,7 +28,9 @@
 
 #pragma once
 
-#include "BHE_Net_ELE_Abstract.h"
+#include <map>
+#include "Eigen/Eigen"
```

Comment:
Eigen is a system include, so it would be `#include <Eigen/Eigen>`

Reply:
ok.